apps/design-land/src/app/loading-icon/loading-icon.component.html

```diff
@@ -1,2 +1,33 @@
-<daff-loading-icon [diameter]="50" color="secondary"></daff-loading-icon>
-<daff-loading-icon [diameter]="50"></daff-loading-icon>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Loading Icon</h1>
+  <p daffArticleLead>
+    The loading icon is used as an indicator of an event in progress.
+  </p>
+
+  <h2>About</h2>
+  <p>
+    This loading icon can be used to indicate that an event is ocurring and is still in progress.
+    The color and size of the loading icon can be customized to suit the needs of your project.
+  </p>
+
+  <h3>Diameter</h3>
+  <design-land-example-viewer example="loading-icon-diameter"></design-land-example-viewer>
+
+  <h3>Theming</h3>
+  <p>
+    The loading icon color can be changed by using the <code>color</code> property. The default color is set to
```

Comment:
The loading icon color is defined by using the <code>color</code> property. By default, the color is set to <code>theme-contrast</code>. This can be changed to one of the supported colors.

Reply:
a140d0cbc6ef94af5282c208eaffde9bdf1fa266